src/websocket/client.ts

```diff
@@ -57,15 +61,9 @@ export class WsClient {
       // errors because that causes nodejs thread to crash
       // with uncaught exception
       socket.onerror = (e) => {
-        if (e.error.code === 'ECONNRESET' || e.error.code === 'ECONNREFUSED') {
-          reject(
-            new Error(
-              `could not connect to holochain conductor, please check that a conductor service is running and available at ${url}`
-            )
-          )
-        } else {
-          reject(e)
-        }
+        reject(new Error(
+          `could not connect to holochain conductor, please check that a conductor service is running and available at ${url}`
```

Comment:
```suggestion
          `error while connecting to holochain at ${url}: ${e.error.name}, ${e.error.message}`
```
Would it be useful to pass an error message to the client? I am not sure.

Reply:
It would be nice, but I think this is what caused the error in the first place.  The error object does not seem to exist in the browser context.  Maybe it does in the node context, but this code throws `Cannot read property 'name' of undefined` so I removed it.

Reply:
ha, interesting